tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -1,40 +1,43 @@
 select distinct ot.*
 from (
   select
-    cd.id personId,
+    cd.id as personId,
     gmc.gmcNumber,
     cd.forenames,
     cd.surname,
     pm1.programmeMembershipType,
     pm1.programmeStartDate,
     pm1.programmeEndDate,
     latestCm.curriculumEndDate,
-    if(prg.programmeName is null and currentPmCounts.count_num > 1, "multiple programmes", prg.programmeName) as programmeName,
-    prg.owner
+    currentPmCounts.programmeNames as programmeName,
+    if(currentPmCounts.count_num > 1, NULL, currentPmCounts.owner) as owner
   from
     ContactDetails cd
   left join GmcDetails gmc on (gmc.id = cd.id)
   left join (
-    -- count current PMs for each person
+    -- count current PMs with combined programme names for each person
     select
-      personId,
+      pm.personId,
+      GROUP_CONCAT(prg.programmeName SEPARATOR " | ") programmeNames,
+      GROUP_CONCAT(prg.owner SEPARATOR " | ") owner,
```

Comment:
If we don't export `owner` here, we will need to left join `Programme` again with the primary search.